File: Basic/13.Find the sum of the digits of a given number/Solution.java
import java.util.Scanner;

public class SumOfDigits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int num = scanner.nextInt();
        int sum = 0;

        while (num != 0) {
            sum += num % 10; // Get the last digit
            num /= 10;       // Remove the last digit
        }

        System.out.println("Sum of the digits: " + sum);
    }
}

File: Basic/13.Find the sum of the digits of a given number/Solution.py
def sum_of_digits(num):
    sum = 0
    while num != 0:
        sum += num % 10  # Get the last digit
        num //= 10       # Remove the last digit
    return sum

if __name__ == "__main__":
    num = int(input("Enter a number: "))
    print(f"Sum of the digits: {sum_of_digits(num)}")
